rebar.config

```diff
@@ -1,19 +1,94 @@
-{erl_opts,
- [
-  {src_dirs, ["src"]},
-  warn_unused_vars,
-  warn_export_all,
-  warn_shadow_vars,
-  warn_unused_import,
-  warn_unused_function,
-  warn_bif_clash,
-  warn_unused_record,
-  warn_deprecated_function,
-  warn_obsolete_guard,
-  strict_validation,
-  warn_export_vars,
-  warn_exported_vars,
-  warn_missing_spec,
-  warn_untyped_record, debug_info
- ]
-}.
+%% -*- mode: erlang;erlang-indent-level: 2;indent-tabs-mode: nil -*-
+%% ex: ts=4 sw=4 ft=erlang et
+
+%% == Erlang Compiler ==
+
+%% Erlang compiler options
+{erl_opts,  [
+          warn_unused_vars,
+          warn_export_all,
+          warn_shadow_vars,
+          warn_unused_import,
+          warn_unused_function,
+          warn_bif_clash,
+          warn_unused_record,
+          warn_deprecated_function,
+          warn_obsolete_guard,
+          strict_validation,
+          warn_export_vars,
+          warn_exported_vars,
+          warn_missing_spec,
+          warn_untyped_record, debug_info
+         ]}.
+
+{profiles, [
+  {test, [
+    {deps, [
+      {zipper, {git, "https://github.com/inaka/zipper.git", {tag, "0.1.4"}}},
+      {katana_test, {git, "https://github.com/inaka/katana-test.git", {tag, "0.0.6"}}},
+      {mixer,       {git, "https://github.com/inaka/mixer.git",       {tag, "0.1.5"}}}
+    ]}
+  ]}
+]}.
+
+%% == Common Test ==
+
+%% {erl_opts, [...]}, but for CT runs
+{ct_compile_opts, [
+                  warn_unused_vars,
+                  warn_export_all,
+                  warn_shadow_vars,
+                  warn_unused_import,
+                  warn_unused_function,
+                  warn_bif_clash,
+                  warn_unused_record,
+                  warn_deprecated_function,
+                  warn_obsolete_guard,
+                  strict_validation,
+                  warn_export_vars,
+                  warn_exported_vars,
+                  warn_missing_spec,
+                  warn_untyped_record, debug_info
+                 ]}.
+
+{ct_opts, []}.
+
+%% == Cover ==
+
+{cover_enabled, true}.
+
+{cover_opts, [verbose]}.
+
+%% == Dependencies ==
+
+{deps, []}.
+
+%% == Dialyzer ==
+
+{dialyzer, [
+    {warnings, [ underspecs
+               , no_return
+               , unmatched_returns
+               , error_handling
+               ]},
+    {get_warnings, true},
+    {plt_apps, top_level_deps},
+    {plt_extra_apps, []},
+    {plt_location, local},
+    {base_plt_apps, [stdlib, kernel]},
+    {base_plt_location, global}
+]}.
+
+%% == Shell ==
+
+{shell, [{apps, [sync]}]}.
+
+%% == xref ==
```

Comment:
We don't usually use `xref` from command line, I think you can remove the xref-related lines from `rebar.config`.
We run xref on the `zipper_meta_SUITE` already :)